### PR TITLE
Code Review of Trial Project

### PR DESCRIPTION
**Pull request by julzedz**
I attempt to review my trial project code and make suggestions applying HTML5 best practices, e.g. Semantic HTML.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<header id="header">
```
Comment (julzedz) [inline index.html:3]: Kindly change the tag from header to head tag. The head tag is the right tag to contain the link tag, the meta tag and the title tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <link rel="stylesheet" href="styles.css">
+  <img id="header-img" src="">Logo</img></br>
+  <nav id="nav-bar">
+    
+      <a href="#features" class="nav-link">Features</a>&nbsp;&nbsp;&nbsp;
+      <a href="#video" class="nav-link"> How it works</a>&nbsp;&nbsp;&nbsp;
+      <a href="#pricing" class="nav-link">Pricing</a>
```
Comment (julzedz) [inline index.html:10]: Kindly enclose the contents of the nav bar in a ul tag. Then, make each item a li tag. The reason is to bring it up to date with the HTML5 best practices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<div id="features">
+<div>Premium materials</div>
+<div>Fast Shipping</div>
+<div> Quality Assurance</div>
+</div></br></br>
+<div id="pricing">
+  <div>$600</div>
+  <div>$900</div>
+  <div>$1200</div>  
+</div>
```
Comment (julzedz) [inline index.html:31]: Kindly make "pricing" div an unordered list instead, and make the children list items. The reason for this is to ensure orderliness and for Search Engine Optimization.

**Review by julzedz — COMMENTED** [review]
# changes required ♻️

Please check my comment below and make all the required changes.

Thanks 🙂

File: index.html (hunk 3 above)
Comment (Qoosim) [reply index.html:31]: - [OPTIONAL] Kindly state the reason for your requests. And if possible point students to any resources that could be of help to the students.

File: index.html (hunk 2 above)
Comment (Qoosim) [reply index.html:10]: - [OPTIONAL] Kindly state the reason for your requests. And if possible point students to any resources that could be of help to the students.

**Review by Qoosim — CHANGES_REQUESTED** [review]
## **Status: Required Changes** :recycle: 

Hi @julzedz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Code Review Performed. :heavy_check_mark: 
-  Good usage of inline comments. :heavy_check_mark: 
-  Good usage of polite comments. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly check if the linters is set up properly and pass. Having no linters set up in a project renders that project invalid and no review would be performed on such project.
- [ ] Kindly update you README file

File: index.html (hunk 3 above)
Comment (julzedz) [reply index.html:31]: Hi @Qoosim, I added linters file, but there were no GitHub action checks. What do I do?

Comment (julzedz) [reply index.html:31]: I can see them now, never mind. Thank you so much @Qoosim 🙏🏾

Comment (julzedz) [reply index.html:31]: @Qoosim I might be wrong but I think this exercise is more about "code-review" than me rewriting the code I wrote during my trial project. Hence, for me to pass the linters check, I will have to start rewriting the trial project code. Do you advise I rewrite the trial code, cuz if I do, the code review I made will be futile?

Comment (julzedz) [reply index.html:31]: To support my stance, kindly see this bullet point from the project requirement 

"Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. (The only difference will be that the Code Reviewer won't check your code but the review you performed - isn't it a code review inception? ;))"

**Review by Lameck1 — COMMENTED** [review]
Hello @julzedz👋,

_This is an additional review as a result of the second opinion request._

I would like to confirm that you are right. @Qoosim, in this task we are not to request changes on the project files but only do a quality check on the review performed by the student. Please take note of this.

Since other requirements are marked as optional and this was the only reason that your project was not approved it will be approved as soon as possible.

Please hold on, you should be able to submit this task on your dashboard ASAP.

Keep rocking 🚀!

**Pull request merged**